Add Peek to apps list

phw opened the pull request:
I recently packaged my app Peek as Flatpak and going forward support both stable and development releases for it.

allanday commented:
Thanks for this! If you could host your flatpakref files elsewhere, we could also include a download button in the click to install section (Github doesn't seem to provide the files as a download). Is that something you'd be interested in doing?

phw commented:
Good points, I moved the flatpakref files to Amazon and it s working now as a download. Not sure about the location for Peek on the download page. I added it to the end just below Picard, as a former Picard developer I feel myself right at home there ;)

allanday commented:
Thanks again for the patches! Let us know if we need to change the links at any point.
